snafu/process.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tools for running subprocesses"""
+from typing import List, TypedDict, Tuple
+import datetime
+import logging
+import subprocess
+
+
+class ProcessRun(TypedDict, total=False):
+    rc: int
+    stdout: str
+    stderr: str
+    runtime: float
```

Comment:
This might be misleading. Meant to represent the time it took to run the process in seconds.